packages/edit-site/src/components/canvas-spinner/index.js

```diff
@@ -1,12 +1,30 @@
 /**
  * WordPress dependencies
  */
-import { Spinner } from '@wordpress/components';
+import { privateApis as blockEditorPrivateApis } from '@wordpress/block-editor';
+import { privateApis as componentsPrivateApis } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+import { useStylesPreviewColors } from '../global-styles/hooks';
+
+const { ProgressBar, Theme } = unlock( componentsPrivateApis );
+const { useGlobalStyle } = unlock( blockEditorPrivateApis );
 
 export default function CanvasSpinner() {
```

Comment:
Probably better to rename folder/components to `CanvasLoader`.

Reply:
Good eye! I've already been planning to do this in a follow-up if you don't mind. I like keeping changes in smaller pieces when possible. Will take care of it as soon as this one gets shipped.

Reply:
I've addressed this in #53728.